rust/lance-index/src/vector/pq.rs

```diff
@@ -228,10 +233,7 @@ impl<T: ArrowFloatType + Cosine + Dot + L2> ProductQuantizerImpl<T> {
         Ok(total_distortion / data.num_rows() as f64)
     }
 
-    /// Pre-compute L2 distance from the query to all code.
-    ///
-    /// It returns the squared L2 distance.
-    fn l2_distance_table(&self, key: &dyn Array, code: &UInt8Array) -> Result<Float32Array> {
+    fn build_l2_distance_table(&self, key: &dyn Array) -> Result<Vec<f32>> {
```

Comment:
Can you add back a comment so we know what this is creating?  Did the meaning of "distance table" change?  It used to be that distance table was distance to code points. Now it looks like it is the distance to pq centroids?

Also, it is a little odd that there is a function named `build_l2_distance_table` and `build_distance_table` and they do not mean the same thing by "distance table".

Maybe rename this method to `build_query_centroid_distances` or something else?

Reply:
This was a sad left-over long time ago. Lemme change these function name for good this time.